modular_skyrat/modules/cargoborg/code/robot_modules.dm

```diff
@@ -16,6 +16,8 @@
 		/obj/item/crowbar/cyborg,
 		/obj/item/extinguisher,
 		/obj/item/universal_scanner,
+		/obj/item/cargo_teleporter,
```

Comment:
If you want to add the cargo teleporter, it should be an upgrade, and not a roundstart thing, otherwise that defeats the intended balance of the cargo borgs.

Reply:
> If you want to add the cargo teleporter, it should be an upgrade, and not a roundstart thing, otherwise that defeats the intended balance of the cargo borgs.

Sure thing.